### PR TITLE
lxc_container: fix the type of the 'container_config' parameter

### PR DESCRIPTION
**Pull request by montag451**
##### SUMMARY
Fix the parameter type of `container_config`.

The documentation of the module states that `container_config` is a list but the type specified in the module code is `str` which generate an annoying warning when `container_config` is used.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
lxc_container

### CODE REVIEW

File: plugins/modules/cloud/lxc/lxc_container.py
Comment (felixfontein) [inline plugins/modules/cloud/lxc/lxc_container.py:1697]: Can you also add `type: list` and `elements: str` to the documentation of `container_config`?

**Comment by Andersson007** [comment]
Any updates on this?

**Comment by cloudnull** [comment]
shipit!

File: plugins/modules/cloud/lxc/lxc_container.py
Comment (montag451) [reply plugins/modules/cloud/lxc/lxc_container.py:1699]: Done in 773ab76

**Comment by felixfontein** [comment]
You still need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to) before this can be merged.

**Comment by montag451** [comment]
@felixfontein, as requested I've added a changelogs fragment for this PR

File: changelogs/fragments/216-fix-lxc-container-container_config-parameter.yaml
Comment (felixfontein) [inline changelogs/fragments/216-fix-lxc-container-container_config-parameter.yaml:2]:
```suggestion
  - lxc_container - fix the type of the ``container_config`` parameter. It is now processed as a list and not a string (https://github.com/ansible-collections/community.general/pull/216).
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
Since @cloudnull approved this PR, I'll merge once CI passes.

**Comment by montag451** [comment]
@felixfontein, do you want me to squash all the commits into one?

**Comment by felixfontein** [comment]
@montag451 no need to. We squash on merge anyway.

**Comment by felixfontein** [comment]
@montag451 thanks for fixing this!
@cloudnull thanks for reviewing!

**Pull request merged**
